src/components/z-status-tag/styles.css

```diff
@@ -0,0 +1,132 @@
+z-status-tag {
+  --z-icon-width: 14px;
+  --z-icon-height: 14px;
+
+  font-family: var(--dashboard-font);
```

Comment:
it's better to use the new, clearer variable `--font-family-sans`

Reply:
risolto